src/JustWatch.elm

```diff
@@ -0,0 +1,216 @@
+module JustWatch exposing (..)
+
+import Json.Decode as Decode exposing (string, list)
+import Json.Decode.Pipeline exposing (decode, hardcoded, optional, required)
+import Http exposing (encodeUri)
+
+
+type MovieOffers
+    = Loading
+    | NoResults
+    | Results (List JustWatchOffer)
+
+
+type OfferType
+    = Buy
+    | Rent
+    | Streaming
+    | Ads
+    | Free
+    | Unknown
+
+
+movieSearchUrl : String -> String
+movieSearchUrl title =
+    "https://apis.justwatch.com/content/titles/en_US/popular?body=" ++ (Http.encodeUri ("{ \"query\": \" " ++ title ++ "\"}"))
+
+
+movieDetailUrl : Int -> String
+movieDetailUrl id =
+    "https://apis.justwatch.com/content/titles/movie/" ++ (toString id) ++ "/locale/en_US"
+
+
+type alias JustWatchSearchResult =
+    { title : String
+    , id : Int
+    }
+
+
+type alias JustWatchSearchResults =
+    { items : List JustWatchSearchResult
+    }
```

Comment:
Also minor, but you might consider dropping this type and just decoding directly to `List JustWatchSearchResult`.  Also goes for `JustWatchDetails` below.